hnn_core/tests/test_dipole.py

```diff
@@ -49,6 +49,12 @@ def test_dipole(tmpdir, run_hnn_core_fixture):
         assert_allclose(dipole_read.data[dpl_key],
                         dipole.data[dpl_key], rtol=0, atol=0.000051)
 
+    # dpls with different scale_applied should not be averaged.
+    with pytest.raises(RuntimeError, match="All dipoles must be scaled equally"
+                       "!"):
```

Comment:
```suggestion
    with pytest.raises(RuntimeError, match="All dipoles must be scaled equally"):
```

just a cosmetic suggestion. Hopefully it fits in 80 chars :)